bpf/lib/nat.h

```diff
@@ -47,11 +47,18 @@ struct nat_entry {
 #define NAT_PUNT_TO_STACK	1
 
 #ifdef HAVE_LRU_MAP_TYPE
-#define NAT_MAP_TYPE BPF_MAP_TYPE_LRU_HASH
+# define NAT_MAP_TYPE BPF_MAP_TYPE_LRU_HASH
```

Comment:
Are these spaces intentional?

Reply:
Yes, it's typically done in order to denote that this define is under the given ifdef; basically kernel coding style.